frontend/src/components/TabbedResults.js

```diff
@@ -116,6 +117,12 @@ class TabbedResults extends Component {
             </Tab>
           );
         })}
+        <Tab key="history" eventKey="history" title="History">
```

Comment:
According to the design this tab should float to the right

Reply:
I didn't move it to right on purpose. The design when we have too many tabs is undefined right now. It would affect implementation a lot.